pkg/mount/nfs.go

```diff
@@ -88,29 +93,34 @@ func (m *nfsMounter) normalizeSource(info *mount.Info, host string) {
 
 // Load mount table
 func (m *nfsMounter) Load(source []string) error {
+	logrus.Trace("Entered Load()")
 	info, err := mount.GetMounts()
 	if err != nil {
 		return err
 	}
 	re := regexp.MustCompile(`,addr=(.*)`)
 MountLoop:
-	for _, v := range info {
+	for i, v := range info {
+		logrus.Trace("Info[%d] = %v", i, *v)
```

Comment:
does this need to be `Tracef` instead? Same at other places

Reply:
Ah yeah, will add as Tracef